dictionary/task_3.py

```diff
@@ -0,0 +1,11 @@
+# Словарь с вложенными данными
+
+books = {
+    "book1": {"title": "1984", "author": "George Orwell", "year": 1949},
+    "book2": {"title": "To Kill a Mockingbird", "author": "Harper Lee", "year": 1960}
+}
+for book_key, book_info in books.items():
+    title = book_info["title"]
+    author = book_info["author"]
+    year = book_info["year"]
+    print(f"Book: {title}, Author: {author}, Year: {year}")
```

Comment:
It is better to get only values in this case because there is no use of `book_key`)

```suggestion
for book_info in books.values():
    title = book_info["title"]
    author = book_info["author"]
    year = book_info["year"]
    print(f"Book: {title}, Author: {author}, Year: {year}")
```